config/config.go

```diff
@@ -53,6 +52,33 @@ func bindFlags(flagSet *pflag.FlagSet) error {
 	return nil
 }
 
+// Resolve ethereum network based on the set boolean flags.
+func (c *Config) resolveEthereumNetwork(flagSet *pflag.FlagSet) error {
+	var err error
+
+	c.Ethereum.Network, err = func() (commonEthereum.Network, error) {
+		isGoerli, err := flagSet.GetBool(commonEthereum.Goerli.String())
+		if err != nil {
+			return commonEthereum.Unknown, err
+		}
+		if isGoerli {
+			return commonEthereum.Goerli, err
+		}
+
+		isDeveloper, err := flagSet.GetBool(commonEthereum.Developer.String())
+		if err != nil {
+			return commonEthereum.Unknown, err
+		}
+		if isDeveloper {
+			return commonEthereum.Developer, err
```

Comment:
I think the common pattern would be returning `nil`
```
return commonEthereum.Developer, nil
```
Same for Goerli network

Reply:
🤦🏻 1b5eaebb4ba5f9117d35765e64c0bd49b822012d